.github/CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 <h3>New features since last release</h3>
 
+* PennyLane now has a `fourier` module, which hosts a growing library 
+  of methods that help with investigating the Fourier representation of functions 
+  implemented by quantum circuits.
+  [(1160)](https://github.com/PennyLaneAI/pennylane/pull/1160)
+  
+  For example, one can plot distributions over Fourier series coefficients like 
+  this one:
+  
+  <img src="https://pennylane.readthedocs.io/en/latest/_images/fourier_vis_violin.png"
+  width=80%></img>
```

Comment:
This looks great!
![image](https://user-images.githubusercontent.com/2959003/119675536-b4dd9580-be6f-11eb-93ed-ad22bc6a3084.png)

Reply:
Added the link!